src/web/mjs/connectors/SeriManga.mjs

```diff
@@ -0,0 +1,66 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class SeriManga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'serimanga';
+        super.label = 'Seri Manga';
+        this.tags = [ 'manga', 'webtoon', 'turkish' ];
+        this.url = 'https://serimanga.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.seri-img div.name');
+        let id = uri.pathname + uri.search;
+        let title = data[0].textContent.trim();
+        return Promise.resolve(new Manga(this, id, title));
```

Comment:
I do not see why this can not be simplified to `return new Manga()`

Reply:
Copy & Paste relict